samples/nrf9160/http_application_fota/Kconfig

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "nRF91 HTTP Client Flash Write example"
```

Comment:
We should use a consistent name for the sample. I used "HTTP application update sample" in the documentation now, which I think is acceptable for "http_application_fota", but we can change it.

Reply:
Agree. I forgot to change "Flash Write" to "Application Fota". I like "HTPP application update" better and will change both the folder name and elsewhere.